xbmc/input/joysticks/JoystickMonitor.cpp

```diff
@@ -22,6 +22,8 @@
 #include "Application.h"
 #include "input/InputManager.h"
 
+#define AXIS_THRESHOLD       0.75f // Axis must exceed this value to be mapped
```

Comment:
No, the threshold is application-dependent. For example, I chose 0.75f in the button mapper because pressure-sensitive buttons send both an analog and a digital button press, and this gives more weight to the digital button press. Here, this threshold determines the sensitivity to awakening the screen saver. 0.75f is a good value, though I don't see any reason it shouldn't be 0.5f.